packages/web/src/components/nav/desktop/PlaylistFolderNavItem.tsx

```diff
@@ -101,76 +100,88 @@ export const PlaylistFolderNavItem = ({
   })
 
   return (
-    <Droppable
-      key={id}
-      className={navColumnStyles.droppable}
-      hoverClassName={navColumnStyles.droppableHover}
-      onDrop={() => {}}
-      acceptedKinds={['library-playlist']}
-    >
-      <FolderNavLink
-        onMouseEnter={() => {
-          setIsHovering(true)
-        }}
-        onMouseLeave={() => setIsHovering(false)}
-        id={id}
-        name={name}
-        onReorder={() => {}}
-        className={cn(navColumnStyles.link, navColumnStyles.editable, {
-          [navColumnStyles.droppableLink]: dragging && isDroppableKind,
-          [navColumnStyles.disabledLink]:
-            dragging && !isDroppableKind && draggingKind !== 'library-playlist'
-        })}
-        onClick={e => {
-          e.preventDefault()
-          e.stopPropagation()
-          setIsExpanded(!isExpanded)
+    <React.Fragment key={id}>
```

Comment:
do we need key here?

Reply:
Yeah because we iterate over the library contents array to render these but maybe I'll move the key up to the component that does the mapping to make it clearer 🤔

Reply:
yeah so key is internal, so just need to apply key to the mapper and should be good!